.github/workflows/system-tests.yml

```diff
@@ -1,14 +1,10 @@
 name: System Tests
 
 on:
-  pull_request:
+  push:
```

Comment:
Use push trigger instead of pull_request.

For some annoying reason github for `${{ github.sha }}` (and the actual commit hash of the repo) will use a merge commit rather than the actual commit which has just been pushed.

This resulted in a mismatch between the test app image tags being published (which used the real commit sha) and the system-tests job here (which used the PR merge commit sha).

To solve this we just use the `push` action for both jobs so that the sha is the same.